state/execution.go

```diff
@@ -155,18 +155,6 @@ func updateValidators(validators *types.ValidatorSet, changedValidators []*abci.
 	return nil
 }
 
-// return a bit array of validators that signed the last commit
-// NOTE: assumes commits have already been authenticated
```

Comment:
we're going to use this soon, so let's not delete it

Reply:
roger

Reply:
still waiting to bring back ?